### PR TITLE
update D00015 / add D00016 CE07

### PR DESCRIPTION
**Pull request by aguirremar**

### CODE REVIEW

**Review by desiderr — APPROVED** [review]
Compared with config.xlsx and cruise report documents. All OK, except that there is a slight discrepancy in the deployment time for ce07shsp_00016:

2024-06-19T17:18:00  pull request and config.xlsx
2024-06-19T17:28:00  cruise report

**Review by cwingard — APPROVED** [review]
Approved. Let's wait until the batch of other PR's are merged, we may need to update this PR for any cross-MIO changes.

**Pull request merged**
